docs/content/Querying.md

```diff
@@ -5,153 +5,50 @@ layout: doc_page
+[Historical](Historical.html), or [Realtime](Realtime.html)). The
 query is expressed in JSON and each of these node types expose the same
-REST query interface.
-
-We start by describing an example query with additional comments that mention
-possible variations. Query operators are also summarized in a table below.
-
-Example Query "rand"
---------------------
-
-Here is the query in the examples/rand subproject (file is query.body), followed by a commented version of the same.
-
-```javascript
-{
-  "queryType": "groupBy",
-  "dataSource": "randSeq",
-  "granularity": "all",
-  "dimensions": [],
-  "aggregations": [
-    { "type": "count", "name": "rows" },
-    { "type": "doubleSum", "fieldName": "events", "name": "e" },
-    { "type": "doubleSum", "fieldName": "outColumn", "name": "randomNumberSum" }
-  ],
-  "postAggregations": [{
-     "type": "arithmetic",
-     "name": "avg_random",
-     "fn": "/",
-     "fields": [
-       { "type": "fieldAccess", "fieldName": "randomNumberSum" },
-       { "type": "fieldAccess", "fieldName": "rows" }
-     ]
-  }],
-  "intervals": ["2012-10-01T00:00/2020-01-01T00"]
-}
-```
-
-This query could be submitted via curl like so (assuming the query object is in a file "query.json").
-
-```
-curl -X POST "http://host:port/druid/v2/?pretty" -H 'content-type: application/json' -d @query.json
-```
-
-The "pretty" query parameter gets the results formatted a bit nicer.
-
-Details of Example Query "rand"
--------------------------------
-
-The queryType JSON field identifies which kind of query operator is to be used, in this case it is groupBy, the most frequently used kind (which corresponds to an internal implementation class GroupByQuery registered as "groupBy"), and it has a set of required fields that are also part of this query. The queryType can also be "search" or "timeBoundary" which have similar or different required fields summarized below:
-
-```javascript
-{
-  "queryType": "groupBy",
-```
-
-The dataSource JSON field shown next identifies where to apply the query. In this case, randSeq corresponds to the examples/rand/rand_realtime.spec file schema:
-
-```javascript
-  "dataSource": "randSeq",
-```
+REST query interface. For normal Druid operations, queries should be issued to the broker nodes.
 
-The granularity JSON field specifies the bucket size for values. It could be a built-in time interval like "second", "minute", "fifteen_minute", "thirty_minute", "hour" or "day". It can also be an expression like `{"type": "period", "period":"PT6m"}` meaning "6 minute buckets". See [Granularities](Granularities.html) for more information on the different options for this field. In this example, it is set to the special value "all" which means bucket all data points together into the same time bucket.
+Although Druid's native query language is JSON over HTTP, many members of the community have contributed different [client libraries](./Libraries.html) in other languages to query Druid.
```

Comment:
I'm not sure what the purpose of the "Query Operators" section is.  Maybe I don't understand what "operator" means in this context.  Would it work to put the two sentences under "Available Queries" (call it just "Queries?") and remove the "Query Operators" section?